examples/oft-solana/programs/oft/src/instructions/init_adapter_oft.rs

```diff
@@ -0,0 +1,66 @@
+use crate::*;
+use anchor_spl::token_interface::{Mint, TokenAccount, TokenInterface};
+
+#[derive(Accounts)]
+#[instruction(params: InitAdapterOftParams)]
+pub struct InitAdapterOft<'info> {
```

Comment:
So you get the adapter and the oft all in one package?  In EVM, we do pure OFT or Adapter as two separate options.  I don't really care, just curious.

Reply:
You need both in one package because you mint accounts off the OFT Program, rather than deploying one or the other.

The OFT Program comes set with:

- OFT (NativeOFT)
- OFT Adapter
- Rate Limits
- Minting

to name a few.